src/main/java/seedu/address/model/item/ItemQuantity.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.item;
+
+import seedu.address.model.tag.Tag;
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class ItemQuantity {
+    public static final String MESSAGE_CONSTRAINTS = "Quantities must be greater than 0, and smaller than 10,000,000.";
+
+    public final int itemQuantity;
```

Comment:
Correct me if I'm mistaken, but since we mentioned before that itemQuantity should be up to an accuracy of 4d.p, should we use a float/double type here instead?

Reply:
Yes

Reply:
Ah yes, thanks for reminding me!